conan/tools/oss/__init__.py

```diff
@@ -0,0 +1,3 @@
+from conan.tools.oss.cross_building import cross_building
```

Comment:
This is not a new namespace at the moment (https://github.com/conan-io/conan/issues/7105)

Put them atm as private inside ``tools.gnu``

Reply:
okay, moved one more time, to `tools.gnu`